### PR TITLE
search: facets search should not trigger reference match

### PR DESCRIPTION
**Pull request by monaawi**
* ref #1136
* Note: we will still not have facets for pub note searches

### CODE REVIEW

File: backend/inspirehep/search/api.py [hunk 1]
```diff
@@ -277,6 +277,11 @@ def get_records_by_pids(pids, source=None, size=10000):
         return results.execute().hits
 
 
+class LiteratureAggregationsSearch(LiteratureSearch):
+    def execute(self, *args, **kwargs):
+        return super(LiteratureSearch, self).execute(*args, **kwargs)
```
Comment (drjova) [inline backend/inspirehep/search/api.py:282]:
```suggestion
        return super().execute(*args, **kwargs)
```

File: backend/tests/integration/search/test_api.py [hunk 2]
```diff
@@ -81,6 +81,40 @@ def test_return_record_for_publication_info_search_with_journal_title_without_do
     assert 200 == response.status_code
 
 
+def test_get_facets_for_publication_info_search(inspire_app):
```
Comment (drjova) [inline backend/tests/integration/search/test_api.py:84]:
```suggestion
def test_facets_for_publication_info_search(inspire_app):
```

File: backend/inspirehep/search/api.py (hunk 1 above)
Comment (monaawi) [reply backend/inspirehep/search/api.py:282]: But I need to call the execute from InspireSearch, not LiteratureSearch.

Comment (drjova) [reply backend/inspirehep/search/api.py:282]: true my bad

**Pull request merged**